src/store/actions.js

```diff
@@ -55,20 +55,40 @@ export default {
     const qualityIds = state.filters.selections.collection_quality ?? collectionQuality
 
     if (qualityIds && qualityIds.length > 0) {
-      api.get(`${COLLECTION_QUALITY_INFO_API_PATH}?attrs=collection(id)&q=assess_level_col=in=(${qualityIds})`).then(response => {
+      let query = ''
+      if (state.filters.satisfyAll.collection_quality) {
+        let tempQuery = ''
+        for (let i = 0; i < qualityIds.length; i++) {
+          tempQuery += 'assess_level_bio' + '==' + qualityIds[i] + ';'
```

Comment:
please use the '@molgenis/rsql' library, it will handle pitfalls like uri encoding

Reply:
Resolved in 
7e86e3f